From 6cf1a9ff11e59c33cb32d3184797e70960ae7fd1 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Thu, 2 Jun 2016 16:56:26 -0500
Subject: [PATCH 20/23] MXSCM-160: ARM: dts: remove duplicated dts files

imx6sxscm-1gb-evb-ldo-lcdif1.dts removed, since it was a
duplicated file for imx6sxscm-1gb-evb-lcdif1-ldo.dts.

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-lcdif1.dts | 36 ----------------------
 1 file changed, 36 deletions(-)
 delete mode 100644 arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-lcdif1.dts

diff --git a/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-lcdif1.dts b/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-lcdif1.dts
deleted file mode 100644
index fd6eaf1..0000000
--- a/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-lcdif1.dts
+++ /dev/null
@@ -1,36 +0,0 @@
-/*
- * Copyright (C) 2014-2015 Freescale Semiconductor, Inc.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6sx-sdb-ldo.dts"
-#include "imx6sxscm-evb.dtsi"
-
-/ {
-	regulators {
-		reg_lcd_3v3: lcd-3v3 {
-			status = "okay";
-		};
-	};
-
-	sii902x_reset: sii902x-reset {
-		status = "okay";
-	};
-};
-
-&csi1 {
-	status = "disabled";
-};
-
-&lcdif1 {
-	status = "okay";
-};
-
-&i2c1 {
-	sii902x@39 {
-		status = "okay";
-	};
-};
-- 
1.9.1

